src/domain/actuator.py

```diff
@@ -0,0 +1,48 @@
+"""Actuator module"""
+
+import datetime
+import logging
+from abc import abstractmethod
+from enum import Enum
+from queue import Queue, Empty
+
+logger = logging.getLogger(__name__)
+
+
+class Actuator:
+    """Base class of an actuator."""
+
+    actuators_pool = {}
+
+    class ActuatorTypes(Enum):
+        RoadSignActuator = "RoadSignActuator"
+        FeederActuator = "FeederActuator"
```

Comment:
Thinking about that, do we really need to defines the actuator types as enum class, and have this saved on the type member of the each instance?

Subclassing should already do that: every subclass "knows" what it's about, and doesn't need a member instance to remind it.